source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -563,14 +538,236 @@ the Kubernetes cluster.
 
          kubectl port-forward service/minio-tenant-1-console 9443:9443
 
-.. _minio-k8s-deploy-minio-tenant-commandline:
 
-Procedure (CLI)
----------------
+Procedure (Command Line)
+------------------------------
 
 The :mc:`kubectl minio tenant create` command supports creating a MinIO Tenant in your Kubernetes cluster.
 The command *requires* that the cluster have a functional MinIO Operator installation.
 
-You can also use this procedure to generate a Kubernetes ``YAML``-formatted resource document for further customization.
+To deploy a tenant from the command line, complete the following steps:
+
+:ref:`create-tenant-cli-determine-settings-required-options`
+
+:ref:`create-tenant-cli-determine-additional-options`
+
+:ref:`create-tenant-cli-enter-command`
+
+:ref:`create-tenant-cli-record-access-info`
+
+:ref:`create-tenant-cli-access-tenant-console`
+
+:ref:`create-tenant-cli-forward-ports`
+
+.. _create-tenant-cli-determine-settings-required-options:
+
+1) Determine Settings for Required Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :mc:`kubectl minio tenant create` command requires several configuration settings.
+Determine the values for all required flags.
+
+For example, the following command creates a new tenant with 
+  
+- the name ``minio_tenant``
+- total capacity of 16 Tebibytes
+- four servers
+- 16 volumes
+- the ``minio`` namespace.
```

Comment:
```suggestion
- in the ``minio`` namespace.
```

Reply:
That doesn't work the the preposition at the end of the introductory statement.
Switching it up and doing a definition list instead.